### ISSUE TITLE
Sometimes translation status incorrect after XLIFF import

### ISSUE THREAD

**Issue by timobrembeck** [digitalfabrik/integreat-cms#1871]
### Describe the Bug

In the production system, we had [the case](https://admin.integreat-app.de/lahndillkreisneu/pages/ru/) that most of the translations were imported as draft, even though the source translations were public.
In theory, the import should [inherit the status from the source translation](https://github.com/digitalfabrik/integreat-cms/blob/3ed0649012dfa9945ea183e35ae401ef0a950413/integreat_cms/xliff/xliff1_serializer.py#L209) if the target translation does not yet exist, so we have to figure out why this didn't work in this case.

### Steps to Reproduce

I wasn't able to reproduce this problem locally yet

### Expected Behavior

If a target translation does not yet exist, it should inherit the status from it source translation.

### Actual Behavior

In some cases, this did not work

### Additional Information

This could be related to recent changes to the translation state: #1613, #1624 or #1691
This might also be an infra ticket, though I have no idea how database inconsistencies could cause this problem.

![Screenshot 2022-11-17 at 12-42-35 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/202437765-39a0d3cd-dd66-466d-872f-c40c9ad5d117.png)

**Comment by MizukiTemma** [comment]
Is German the direct source language of Russian in the reported case in the foto?
If not (Russian has another language X as source language and X has German as cource language), one possible scenario was found.

Reproducing is possible for example with Arabic as target language, while Arabic has English as source language and English has German as source language.

1. Create a new page in German and publish this.
2. Create a xliff file for arabic translation.
3. Import this xliff.
4. The status of the arabic translation is draft, whereas the german translation is public.

Users cannot export xliff file for arabic translation when there is no english (direct source language) translation yet. Though users can of course create an xliff file by themselves (for exmaple, by modifing page number and language code in a xliff file, which was originally exported for another language or another language).

**Comment by timoludwig** [comment]
> Is German the direct source language of Russian in the reported case in the photo?

Yes, the region only has one source language (German) and all other languages directly depend on it.
Thanks a lot for digging into this issue, but unfortunately there seems to be a different problem...

**Comment by MizukiTemma** [comment]
Thank you for quick reply 😃 
Then I can look for another possible case 👍

**Comment by osmers** [comment]
@timoludwig is there still a question attached to this issue?

**Comment by osmers** [comment]
@timobrembeck do you need any more info for this issue?

**Comment by timobrembeck** [comment]
@osmers Yes, but we don't know what information that is. We were unable to reproduce the issue, so we don't know what we're supposed to do here.
If this happened/happens again, please let us know as much about it as possible.

**Comment by osmers** [comment]
@timobrembeck I asked the team and apparently there is no consistency or regularity to it, it just happens randomly. So I put the issue into the backlog and if we find some rule as to when and why it happens, we'll put it in here.

**Comment by svenseeberg** [comment]
It seems that automatic translations are also affected sometimes.

**Comment by svenseeberg** [comment]
```
integreat=# SELECT version, minor_edit, id, status, machine_translated,automatic_translation,last_updated FROM cms_pagetranslation WHERE page_id=13358 AND language_id IN (3,56) ORDER BY last_updated;
 version | minor_edit |   id    |  status   | machine_translated | automatic_translation |         last_updated          
---------+------------+---------+-----------+--------------------+-----------------------+-------------------------------
[...]
      24 | f          | 5407041 | PUBLIC    | f                  | t                     | 2023-11-11 10:44:49.678095+01
       2 | f          | 5407050 | PUBLIC    | t                  | f                     | 2023-11-11 10:44:56.434069+01
      25 | f          | 5407142 | PUBLIC    | f                  | t                     | 2023-11-13 08:59:36.528548+01
       3 | f          | 5407151 | PUBLIC    | t                  | f                     | 2023-11-13 08:59:44.45937+01
      26 | f          | 5407152 | PUBLIC    | f                  | t                     | 2023-11-13 08:59:58.326608+01
       4 | f          | 5407161 | PUBLIC    | t                  | f                     | 2023-11-13 09:00:05.376893+01
```

**Comment by PeterNerlich** [comment]
Are we aware of this happening still in the last year? @laravoelk 

@juliankehne If you would be able to tease out some more specific conditions under which this happens, we would be thankful! *(this is not a high priority for us)*

**Comment by laravoelk** [comment]
I got some infos from the team and this has not happened in the last year.

**Comment by juliankehne** [comment]
@PeterNerlich I wasn't able to reproduce it.
